tests/ha/single_pod_failure/test_server_pod_failure.py

```diff
@@ -201,6 +220,7 @@ def test_degraded_reads_safe_server_pod_shutdown(self):
     @pytest.mark.ha
     @pytest.mark.lc
     @pytest.mark.tags("TEST-39903")
+    @pytest.mark.skip("Skipped until CORTX-33759 is fixed")
```

Comment:
is this require?

Reply:
yes this we would keep for now for CFT regression cycle but then once we implement some other method, we might either remove the test itself or modify this one with diff method